codechain/config/presets/config.prod.toml

```diff
@@ -3,6 +3,7 @@ quiet = false
 base_path = "."
 chain = "mainnet"
 
+
```

Comment:
Isn't it an unnecessary change?

Reply:
Oh, I remember that I reverted it. But it has inserted in someway again!
I removed it again.